src/testing/mission.test.js

```diff
@@ -0,0 +1,46 @@
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import Mission from '../components/Mission';
+
+const mockStore = configureStore([]);
+const dummyData = [
+  {
+    mission_id: '1',
+    mission_name: 'Test Mission 1',
+    description: 'Test Mission 1 description',
+    reserved: false,
+  },
+  {
+    mission_id: '2',
+    mission_name: 'Test Mission 2',
+    description: 'Test Mission 2 description',
+    reserved: false,
+  },
+];
+
+describe('Mission component', () => {
+  let store;
+
+  beforeEach(() => {
+    store = mockStore({
+      mission: {
+        mission: dummyData,
+        hasFetched: true,
+      },
+    });
+  });
+
+  test('should render the first mission', () => {
+    const { getByTestId } = render(
+      <Provider store={store}>
+        <Mission />
+      </Provider>,
+    );
+
+    const firstMission = getByTestId('1');
+
+    expect(firstMission).toHaveTextContent('Test Mission 1');
+  });
+});
```

Comment:
- [ ] **Hello, I would advise you to add a unit test for Rocket. Because it is one of the key elements of your project.**
-
![Capture](https://github.com/sagieramos/space_traveler_hub/assets/63972670/9779705a-75de-4810-a45d-30e3d818b6ef)